pkg/storage/test/storage.go

```diff
@@ -53,3 +56,34 @@ func RunAllTests(t *testing.T, ds storage.OpenFGADatastore) {
 	// Stores.
 	t.Run("TestStore", func(t *testing.T) { StoreTest(t, ds) })
 }
+
+// BootstrapFGAStore is a utility which can used to quickly bootstrap an OpenFGADatastore
+// with an FGA model and relationship tuples. It returns the (store_id, model_id), respectively.
+func BootstrapFGAStore(
```

Comment:
Bit tangential, but added this as per prior conversations and to assist with making these kinds of changes easier next time around.